wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WDateFieldWebElementTest.java

```diff
@@ -0,0 +1,43 @@
+package com.github.bordertech.wcomponents.test;
+
+import com.github.bordertech.wcomponents.test.components.WDateFieldUI;
+import com.github.bordertech.wcomponents.test.selenium.ByButton;
+import com.github.bordertech.wcomponents.test.selenium.ByLabel;
+import com.github.bordertech.wcomponents.test.selenium.MultiBrowserRunner;
+import com.github.bordertech.wcomponents.test.selenium.SeleniumJettyTestCase;
+import com.github.bordertech.wcomponents.test.selenium.driver.SeleniumWComponentsWebDriver;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWButtonWebElement;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWDateFieldWebElement;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWFieldIndicatorWebElement;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MultiBrowserRunner.class)
+public class WDateFieldWebElementTest extends SeleniumJettyTestCase {
+
+
+	/**
+	 * Constructor...
+	 */
+	public WDateFieldWebElementTest() {
+		super(new WDateFieldUI());
+	}
+
+	@Test
+	public void testDateFieldUsingTextFeatures() {
```

Comment:
This is good so far as it goes but the new WebElement extension should be tested getting date fields in the various render states (with and without preset date content,  and in disabled and read-only states with and without date content).